src/Umbraco.Web.UI.NetCore/umbraco/PartialViewMacros/Templates/EditProfile.cshtml

```diff
@@ -1,14 +1,12 @@
-﻿@using System.Web.Mvc.Html
-@using Umbraco.Web
-@using Umbraco.Web.Composing
-@using Umbraco.Web.Controllers
-@inherits Umbraco.Web.Macros.PartialViewMacroPage
+﻿@using Umbraco.Core.Security
+@using Umbraco.Extensions
+@using Umbraco.Web.Website.Controllers
+@inherits Umbraco.Web.Common.Macros.PartialViewMacroPage
+@inject IUmbracoWebsiteSecurityAccessor UmbracoWebsiteSecurityAccessor
 
 @{
-    var profileModel = Current.MembershipHelper.GetCurrentMemberProfileModel();
+    var profileModel = await UmbracoWebsiteSecurityAccessor.WebsiteSecurity.GetCurrentMemberProfileModelAsync();
```

Comment:
I think we should save `UmbracoWebsiteSecurityAccessor.WebsiteSecurity` in a variable